lighthouse-core/test/audits/third-party-facades-test.js

```diff
@@ -0,0 +1,450 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const ThirdPartyFacades = require('../../audits/third-party-facades.js');
+const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
+const createTestTrace = require('../create-test-trace.js');
+
+const pwaTrace = require('../fixtures/traces/progressive-app-m60.json');
+const pwaDevtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+const videoEmbedsTrace = require('../fixtures/traces/video-embeds-m84.json');
+const videoEmbedsDevtolsLog = require('../fixtures/traces/video-embeds-m84.devtools.log.json');
+const noThirdPartyTrace = require('../fixtures/traces/no-tracingstarted-m74.json');
+
+function resourceEntry(startTime, headerEndTime, endTime, transferSize, url) {
```

Comment:
FWIW, I've come to regret *almost* every test helper API I've made where there are no names whatsoever 

I'm not asking to change it, but from someone who has written many of these in LH, just a nudge of personal experience to consider switching sides and consider using sensible defaults but keep the object names instead 😁

e.g. derive headerEndTime as startTime +1 for all the tests that don't care, derive endTime as startTime +2, pass in `{startTime: 100, url: '...'}`

<img src="https://user-images.githubusercontent.com/2301202/98748716-d36eba00-237f-11eb-9f39-e8fed069da09.png" width="200">

Reply:
The biggest reason I made the test helpers was to make the request timing easy to understand. Now that request timing doesn't matter anymore, I think this change is worth making.